nixos/modules/services/games/factorio.nix

```diff
@@ -136,6 +136,13 @@ in
           derivations via nixos-channel. Until then, this is for experts only.
         '';
       };
+      mods-dat = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        description = lib.mdDoc ''
+        Mods settings can be changed by specifying a dat file.
```

Comment:
Indent slightly more, and if possible include a link to what this is. It's not obvious from reading this description.